Scripts/Source/minai_AmbientSexTalk.psc

```diff
@@ -0,0 +1,118 @@
+Scriptname minai_AmbientSexTalk extends Quest 
+
+minai_Config config
+minai_Sex sex
+minai_MainQuestController main
+
+SexLabFramework slf = None
+; store registered threads by ostim/sexlab framework:  {ostim: [0,1,2,3], sexlab: [0,1,2,3]}
+int jThreadsByFrameworkMap
+int playerThread = -1
+
+function Maintenance(minai_Sex _sex, SexLabFramework _slf)
```

Comment:
Overall looks good - Let's add a failsafe here to cleanup threads on game load as well to make sure nothing gets stuck in a bad state from sex scenes breaking somehow.

Reply:
I added `releaseAndRetain` on load